index.html

```diff
@@ -5167,6 +5167,61 @@ <h2>The <code>application/vc+ld+json</code> Media Type</h2>
   <p>A [[JSON-LD]] context is expected to be present in the body of the document, and as indicated by the presence of `ld+json` in the media type, the credential is expected to be a valid <a href="https://www.w3.org/TR/json-ld11/#dfn-json-ld-document">JSON-LD document</a>.</p>
 </section>
 
+<section id="vp-ld-media-type">
+  <h2>application/vp+ld+json</h2>
+  <p>
+    This specification registers the <code>application/vp+ld+json</code> Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Presentations format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>vp+ld+json</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        Resources that use the "<code>application/vp+ld+json</code>"
+        Media Type are required to conform to all of the requirements
+        for the "<code>application/ld+json</code>" Media Type and are
+        therefore subject to the same encoding considerations specified
+        in Section 11 of [[RFC7159]].
+      </td>
+    </tr>
+    <tr>
+      <td>Security considerations: </td>
+      <td>As defined in this specification.</td>
+    </tr>
+    <tr>
+      <td>Contact: </td>
+      <td>
+        W3C Verifiable Credentials Working Group <a href="mailto:public-vc-wg@w3.org">public-vc-wg@w3.org</a>
+      </td>
+    </tr>
+  </table>
+
+  <p>
+    Note that while the Verifiable Credentials format uses JSON-LD
+    conventions, there are a number of constraints and additional
+    requirements for Verifiable Credential implementations that justify
+    the use of a specific media type.
+  </p>
+
+  <p>
+    This media type can be used with credentials secured using an <a>external proof</a>.
```

Comment:
```suggestion
    This media type can be used with credentials secured using an <a>external proof</a>.
```

Hmm, this looks wrong? Is this what we're trying to say? I'm going to merge this, but wondering if this is a copy-paste error. If so, @OR13, can you fix it in a future (editorial) PR?